gpii/node_modules/flowManager/src/UntrustedSettingsDataSource.js

```diff
@@ -29,10 +29,40 @@ fluid.require("%gpii-universal/gpii/node_modules/gpii-oauth2/gpii-oauth2-utiliti
 fluid.defaults("gpii.flowManager.untrustedSettingsDataSource", {
     gradeNames: ["fluid.component"],
 
-    // Options that are distributed down from, e.g., gpii.flowManager.untrusted.config.development
-    untrustedSettingsGetUrl: null,
-    untrustedSettingsPutUrl: null,
-    accessTokenUrl: null,
+    // This option should be distributed down from, e.g., gpii.flowManager.untrusted.config.development
+    flowManagerBaseUrl: null,
+
+    untrustedSettingsGetUrlTemplate: "%flowManagerBaseUrl/%gpiiKey/untrusted-settings/%device",
+    untrustedSettingsPutUrlTemplate: "%flowManagerBaseUrl/%gpiiKey/untrusted-settings",
+    accessTokenUrlTemplate: "%flowManagerBaseUrl/access_token",
+
+    // The URLs below are calculated from the flow manager base URL and their templates.
+    // These URLs are in the "options" block instead of "members" block is because distributeOptions source
```

Comment:
Not sure about this comment - if you wanted to distribute to a member you would distribute to options.members - but isn't this the most natural place for them anyway?